rust/chains/hyperlane-cosmos/src/libs/verify.rs

```diff
@@ -0,0 +1,131 @@
+use std::cmp;
+
+use bech32::{FromBase32, ToBase32};
+use cosmrs::crypto::secp256k1::SigningKey;
+use hyperlane_core::{ChainCommunicationError, ChainResult, H160, H256};
+use ripemd::Ripemd160;
+use sha2::{Digest, Sha256};
+
+/// decode bech32 address to H256
+pub fn bech32_decode(addr: String) -> H256 {
+    let (_hrp, data, _variant) = bech32::decode(addr.as_str()).unwrap();
+
+    let value = Vec::<u8>::from_base32(&data).unwrap();
+    let mut result: [u8; 32] = [0; 32];
```

Comment:
would be nice to remove these magic numbers here and below. Can you use a constant imported from one of the crates at the top of the file for the `32`, and also just assign `Default::default()` here?